projects/pistache/fuzz_parsers.cpp

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2021 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include <pistache/http.h>
+
+using namespace Pistache;
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    std::string input(reinterpret_cast<const char*>(data), size);
+
+    // URI parsing
+    Http::Uri::Query query1;
```

Comment:
shall we create a different fuzzing target for uri?

Reply:
I think I will keep it in here now and then monitor coverage of the target - if we get don't get proper coverage we can split it up